src/Messages.js

```diff
@@ -363,12 +363,23 @@ export default defineMessages({
     description: 'Risk of Change',
     defaultMessage: 'Risk of change',
   },
+  riskOfChange: {
+    id: 'riskOfChange',
+    description: 'Risk of Change',
+    defaultMessage: 'Risk of change',
+  },
   rulesDetailsTotalriskBody: {
     id: 'rulesdetails.totalriskbody',
     description: 'Text explaining the total risk value of this recommendation',
     defaultMessage: `The total risk of this remediation is <strong>{risk}</strong>,
                         based on the combination of likelihood and impact to remediate.`,
   },
+  rulesDetailsTotalRiskBody: {
```

Comment:
For rulesDetailsTotalRiskyBody and riskOfChange, I understand changing to camelCase but im unsure why have duplicates instead of replacing the originals and doing something like a global find/replace?

Reply:
Yep, good point. I left it for future work, but seems reasonable to do it already within this patch :+1 will push extra commit soon